Deployment/VMware vSphere/Standalone/Large/variables.tf

```diff
@@ -11,52 +11,44 @@ variable "vsphere-user" {
 variable "vsphere-password" {
   type        = string
   description = "VMWare vSphere admin user password"
-  default     = "<password>"
 # Turn on for production use. Will hide the entire custom variable entry
 #  sensitive   = true
```

Comment:
Same as above, this should probably be uncommented always and the comment above it removed.